src/wavm/CMakeLists.txt

```diff
@@ -34,5 +34,6 @@ set(LIB_FILES
         )
 
 faasm_private_lib(wavmmodule "${LIB_FILES}")
+target_compile_options(wavmmodule PRIVATE "-D__wasm__")
```

Comment:
I am not sure why the `define` in `wasi-libc` does not get here, but it does not. I am also concerned that other compile options we pass in `cmake/ExternalProjects.cmake` like `WAVM_ENABLE_UNWIND=ON` are not passed to the compiler when _actually_ building `WAVM`.

Reply:
We're not building this to wasm, so why do you expect `-D__wasm__` to be defined?

Reply:
I keep getting confused with this. Thanks.

Wrt `WAVM_ENABLE_UNWIND` it is set to ON by default [here](https://github.com/faasm/WAVM/blob/95fd0c4374f68570cfd2a7e51b227bf593d5eec7/CMakeLists.txt#L129), so I think there's no need to pass the extra argument.

Reply:
Ok great, so we can remove this definition right?